frontend/projects/ui/src/app/modals/generic-input/generic-input.component.module.ts

```diff
@@ -14,6 +15,7 @@ import { FormsModule } from '@angular/forms'
     FormsModule,
     RouterModule.forChild([]),
     SharedPipesModule,
+    MaskPipeModule,
```

Comment:
Pretty sure you don't need that when you provide and inject pipe like a service.